package-manifest.toml

```diff
@@ -318,8 +318,8 @@ only_for_targets.image = "standard"
 # 2. Copy dendrite.tar.gz from dendrite/out to omicron/out
 source.type = "prebuilt"
 source.repo = "dendrite"
-source.commit = "6de087190ed4afa7ffbcede51140a7eff4cc5a60"
-source.sha256 = "4576e3e8b5eb83b9fb8d3d867998bd25f6a9f1f30c92868bf745afa46e4e6926"
+source.commit = "51fb973674dfc1c372621bd83970c147684a005f"
```

Comment:
Point to commit on `main` once https://github.com/oxidecomputer/dendrite/pull/448 is merged. Ditto for others below.

Reply:
Done in 062e6962